Makefile

```diff
@@ -0,0 +1,43 @@
+KUEUE_REPO=https://github.com/kubernetes-sigs/kueue.git
+KUEUE_TMP_PATH=/tmp/xpk_tmp/kueue
```

Comment:
I would propose to use BIN_PATH=./bin where we have all bin files and add it to `$PATH`.

Reply:
We git clone to tmp, build it there and than copy binary to ~/.local/bin/ to avoid using sudo

Reply:
Ah, sorry. So we can create separate variable to save all binaries.